Tests/SwiftStdlibTests/OptionalExtensionsTests.swift

```diff
@@ -63,6 +63,24 @@ final class OptionalExtensionsTests: XCTestCase {
 		XCTAssert(parameters[key2] == parameter2)
 	}
 
+    func testConditionalAssignment() {
+        var text1: String?
+        text1 ?= "newText1"
+        XCTAssert(text1 == "newText1")
```

Comment:
I think it's cleaner to use `XCTAssertEquals`

Reply:
fixed